packages/altair-core/src/types/state/settings.interfaces.ts

```diff
@@ -106,6 +106,11 @@ export interface SettingsState {
    */
   disableLineNumbers?: boolean;
 
+  /**
+   * Hides deprecated Doc items
+   */
+  hideDeprecatedDocItems?: boolean;
```

Comment:
can we make this something like `doc.hideDeprecatdeItems`? Trying to scope the settings to more relevant namespaces

Reply:
Yes, makes sense. I will make the change in the next revision.